base/base_tests/small_set_test.cpp

```diff
@@ -14,12 +14,6 @@ namespace small_set_test
 {
 using namespace base;
 
-UNIT_TEST(SmallSet_Empty)
```

Comment:
Why is it removed?

Reply:
`SmallSet<0>` creates an array of zero size: `uint64_t m_blocks[0] = {};` which produces a warning.
I added an assert to check this behavior: `static_assert(kNumBlocks > 0);`
There is no need for this test now.